changelog/fragments/enable-seccompprofile-run-bundle.yaml

```diff
@@ -0,0 +1,18 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Added --security-context-config flag to enable seccompprofile.
```

Comment:
Now that I think about it, it might be nice to add the prefix `operator-sdk run bundle(-upgrade):` so that when it shows up in the changelog users know what command this impacts.

Reply:
Good point, updated.